Automattic/jetpack
# CRM: Add business telephone to invoices

This PR adds the business telephone number to the invoices.

## Proposed changes:

Add the business telephone number to the invoices.

### Other information:


## Jetpack product discussion


https://github.com/Automattic/zero-bs-crm/issues/2757

## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Configure a telephone number in the business settings (`wp-admin/admin.php?page=zerobscrm-plugin-settings&tab=bizinfo`)
* Create an invoice (`wp-admin/admin.php?page=zbs-add-edit&action=edit&zbstype=invoice`)
* Test all three rendering options for this invoice (located in the invoice edit page - e.g. `wp-admin/admin.php?page=zbs-add-edit&action=edit&zbstype=invoice&zbsid=4`)
![image](https://user-images.githubusercontent.com/37049295/227017679-05fc3b17-2cbc-47bf-bc84-a56c8578e28f.png)
* In `trunk` the telephone is not shown in invoices.
![image](https://user-images.githubusercontent.com/37049295/227019127-82e3b92a-62bc-45cf-aaa0-b3a553ae7b28.png)
* In `fix/crm-business-telephone-not-shown` the telephone is shown in invoices.
![image](https://user-images.githubusercontent.com/37049295/227019061-06d0eb0d-a3cd-4e41-8c91-4ebf45b7aefe.png)

## NOTE
After implementing this new feature we stumbled across the field `Extra Invoice Info:` in the invoice settings page that could achieve the same thing.
In `trunk`:
![image](https://user-images.githubusercontent.com/37049295/227019864-07762f17-48d8-4f65-ac88-9b3ea361891a.png)

![image](https://user-images.githubusercontent.com/37049295/227020043-4c06f200-1502-4621-ae5f-d0afed2b25c6.png)

Should we even go ahead merging this PR?
With this PR some people that didn't want their number to be public in the first place will have it displayed in their invoices. By not merging and putting that information about the extra invoice info in the KB we are giving them a choice of what will be displayed without adding any more settings (e.g. `Display phone number in invoices [ ]`).
